python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -211,17 +214,21 @@ def evaluate(self, *, cache: MutableMapping[int, DataFrame]) -> DataFrame:
         with_columns = options.with_columns
         row_index = options.row_index
         if self.typ == "csv":
+            opts, cloud_opts = map(json.loads, self.options)
```

Comment:
Is it expected that we don't need to do anything with the new options? I'm not surprised that we're ignoring cloud options.

Reply:
It is a bug that we are doing nothing with them. But previously we were also doing nothing with them